x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -89,9 +90,14 @@ export const createPackagePolicyHandler: RequestHandler<
   const user = appContextService.getSecurity()?.authc.getCurrentUser(request) || undefined;
   const { force, ...newPolicy } = request.body;
   try {
+    const newPackagePolicy = await packagePolicyService.enrichPolicyWithDefaultsFromPackage(
```

Comment:
this new enrich function has a side effect of querying `epm registry` even if all data is provided in request body. could that be a problem?

Reply:
I think it's okay we already query the registry later on a package policy creation when we compile the template for the agent.